server/data-import.js

```diff
@@ -87,21 +83,18 @@ let readWriteAnswerPhotos = async () => {
         })
       })
   }
-  catch (err) {
-    console.log('err in data-import answer photos', err)
-  }
+  catch (err) { return err.message }
 }
 
 
 // initial load of data
-let initialLoad = async () => { await readWriteQuestions() }
-let secondLoad = async () => { await readWriteAnswers() }
-let thirdLoad = async () => { await readWriteAnswerPhotos() }
-
 let loadAllThree = async () => {
-  await initialLoad()
-  await secondLoad()
-  await thirdLoad()
+  await readWriteQuestions()
+  await readWriteAnswers()
+  await readWriteAnswerPhotos()
+
+  console.log('start update counts')
+  await db.updateCounts()
 }
```

Comment:
This file looks super clean, nicely done and thank you for your help cleaning mine up!